web/src/components/navbar/BusinessNjGovLogo.tsx

```diff
@@ -0,0 +1,31 @@
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import analytics from "@/lib/utils/analytics";
+import { useRouter } from "next/router";
+import { ReactElement } from "react";
+interface Props {
+  useSmall?: boolean;
+}
+
+export const BusinessNjGovLogo = (props: Props): ReactElement => {
```

Comment:
pebble/dust - So is this only for the nav bar? It kinda seems like it based on how the onClicks are handled. Maybe should be named as such

Reply:
named it `NavbarBusinessNjGovLogo.tsx` to match the style of the other navbar components